README.md

```diff
@@ -189,7 +189,7 @@ You can also add languages after initializing:
 const humanizer = humanizeDuration.humanizer()
 
 humanizer.languages.shortEn = {
-  y: function(c) { return c + 'y' },
+  y: () => `${c}y`,
```

Comment:
Does this need the `${c}` part?

Reply:
it’s the standard way for string concarenation in es6

Reply:
fixed